### PR TITLE
please to verif it

### PR DESCRIPTION
**Pull request by jeremylzn**

### CODE REVIEW

**Comment by jeremylzn** [comment]
minor update design

**Comment by jeremylzn** [comment]
minor

File: frontend/style/app/manage.css [hunk 1]
```diff
@@ -14,27 +14,45 @@
...
-    padding: 10px;
-    background-color: white;
+    border-radius: 5%/4%;
+    padding: 2%;
+    background-color: rgb(211, 230, 241);
+    border: 2px solid #00bfff;
 }
+
+
+select.swal2-input{
```
Comment (SamuelBismuth) [inline frontend/style/app/manage.css]: Here you have to define another class with a new class name,  please don't override an existing class...

File: frontend/style/app/manage.css [hunk 2]
```diff
@@ -14,27 +14,45 @@
...
+    /* just "box-shadow" it's not possible to convert to % */
+    -webkit-box-shadow: inset 1px 1px 1px rgb(189, 184, 184);
+    -moz-box-shadow: inset 1px 1px 1px rgb(189, 184, 184);
+    box-shadow: inset 1px 1px 1px rgb(189, 184, 184);
+    overflow: auto;   
+    border: 1px solid #aaa;
+}
+
+textarea.swal2-input{
```
Comment (SamuelBismuth) [inline frontend/style/app/manage.css]: Same thing here.

File: frontend/style/app/home.css [hunk 3]
```diff
@@ -80,18 +80,26 @@
...
+  /* just "box-shadow" it's not possible to convert to % */
+  box-shadow: 5px 10px 18px grey;
+  border-radius: 5%/4%;
+  padding: 2%;
+  background-color: white;
+}
+
+/* Font of the exercise in the home page */
+
+.text-register{
```
Comment (SamuelBismuth) [inline frontend/style/app/home.css]: Do you think that we need to reuse in multiple file? If yes, we need to put this in the thirdParty file.

File: frontend/style/app/home.css [hunk 4]
```diff
@@ -80,18 +80,26 @@
...
+    font-family: monospace;
+}
+
+.timestamp{
```
Comment (SamuelBismuth) [inline frontend/style/app/home.css]: We need to reuse the timestamp in multiple file we need to put this in the thirdParty file.

File: frontend/style/button.css [hunk 5]
```diff
@@ -15,16 +15,21 @@
   -o-text-overflow: clip;
```
Comment (SamuelBismuth) [inline frontend/style/button.css]: Good job here.

File: frontend/util/html/accordion.js [hunk 6]
```diff
@@ -36,15 +36,15 @@ function createAccordionBodyHomeRegister(exercise) {
 		'<div class="exercise" name="Vu1XBFXwv7aXLWnWuTADwBUOzQD2_1">' +
```
Comment (SamuelBismuth) [inline frontend/util/html/accordion.js]: Great

**Comment by jeremylzn** [comment]
final manage and home page css

**Pull request merged**